klock/src/commonTest/kotlin/com/soywiz/klock/locale/KlockLocaleTest.kt

```diff
@@ -276,4 +276,17 @@ class KlockLocaleTest {
 			HttpDate.withTimezoneNames(ExtendedTimezoneNames).parse("Tue, 19 Sep 2017 00:58:45 MSK").toString()
 		)
 	}
+
+    @Test
+    fun testOrdinalsEnglish() {
```

Comment:
I think better to test the 31 days at once: it is failing for 11, 12 and 13. In fact, the code I put was incomplete since those are irregulars.

Reply:
Done

Reply:
I usually prefer using string literals and joining to string so you can copy paste the results when changed.
Still this should do